File: app/src/main/java/com/polok/eubmanagement/presentation/SplashActivity.java
package com.polok.eubmanagement.presentation;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;
import com.polok.eubmanagement.R;
import com.polok.eubmanagement.presentation.signin.SignInActivity;
import com.polok.eubmanagement.util.Extension;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Extension.hideStatusBar(getWindow());
        setContentView(R.layout.activity_splash);

        new Handler(Looper.myLooper()).postDelayed(new Runnable() {
            @Override
            public void run() {
                startActivity(new Intent(SplashActivity.this, SignInActivity.class));
                finish();
            }
        },2000);
    }
}

File: app/src/main/java/com/polok/eubmanagement/presentation/SplashActivity.kt
package com.polok.eubmanagement.presentation

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.polok.eubmanagement.R
import com.polok.eubmanagement.presentation.auth.AuthActivity

@SuppressLint("CustomSplashScreen")
class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        window?.decorView?.postDelayed({
            startActivity(Intent(this, AuthActivity::class.java))
            finish()
        }, 2000)
    }
}